packages/nuxt/src/core/nuxt.ts

```diff
@@ -369,15 +369,16 @@ async function initNuxt (nuxt: Nuxt) {
 
   addModuleTranspiles()
 
-  // Init nitro
-  await initNitro(nuxt)
-
   // TODO: remove when app manifest support is landed in https://github.com/nuxt/nuxt/pull/21641
   // Add prerender payload support
-  if (useNitro().options.static && nuxt.options.experimental.payloadExtraction === undefined) {
+  if (nuxt.options.nitro?.static && nuxt.options.experimental.payloadExtraction === undefined) {
```

Comment:
I think we need to find another way to resolve. Nitro static presets need static behaviour even though this won't be set in the Nuxt options.

Reply:
Oh, I see. Then let's chat about that 👍🏻